Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.0.9-fpm-alpine3.14
+FROM php:8.0-fpm-alpine
```

Comment:
I think having the versions set here would be better

Reply:
I tried a clone from @dotalchemy but I still get the "permission denied" after setting up the containers. I assume there is an permission issue inside the app or nginx container. I just can't find it. I've tried to track this down for several months now :p

`The stream or file "/var/www/storage/logs/laravel-2022-01-12.log" could not be opened in append mode: Failed to open stream: Permission denied`

Reply:
Would a final chmod before starting help? Or would that totally mess up the layer caching?

Reply:
For specific versions, my concern there is that the 8.0.x releases of PHP are bug / security fixes - if we think about phpvms as a "production" service, as someone deploying it I'd like to make sure that my images have the latest point patches in them.

For PHP, I definitely wouldn't use the `php:8-fpm-alpine` tag because then you're gonna end up with PHP 8.1, which is a feature release, but with `php:8.0-fpm-alpine` then I'm going to get the bug and security fixes up to 8.0.14 without alteration / new features introduced into 8.1 - https://www.php.net/ChangeLog-8.php

The same with Alpine, though that's coming from PHP in this case, so one would _assume_ whatever Alpine release under 8.0.x has been tested appropriately prior to tagging.

I think is a safe balance between ensuring the container is secured and bug-free at the same time as avoiding the risk of an `8.x` tag.

Reply:
> `The stream or file "/var/www/storage/logs/laravel-2022-01-12.log" could not be opened in append mode: Failed to open stream: Permission denied`

Can you do `docker exec -it containername /bin/sh` to get a shell then `ls -la` on `/var/www/storage/` and `/var/www/storage/logs`